Add Edda Keyboard (#13953)
qmk/qmk_firmware: keyboards/edda/readme.md

# Edda PCB

Alice layout PCB with top mounted USB C used in Mekanisk Edda

* Keyboard Maintainer: [martin](https://github.com/arnstadm)
* Hardware Supported: [Edda PCB](https://tastatur.no/collections/forhandsbestilling-1/products/edda-pcb?variant=39481626001453)

Make example for this keyboard (after setting up your build environment):

>     make edda:default:flash
> 
> For VIA support
> 
>     make edda:via:flash

Reviewer: ```suggestion
    make edda:default

Flashing example for this keyboard:

    make edda:default:flash
```

> Edda PCB
> ======
> 
> Alice layout PCB with top mounted USB C used in Mekanisk Edda
> 
> Keyboard Maintainer: [martin](https://github.com/arnstadm)  
> Hardware Supported: [Edda PCB](https://tastatur.no/collections/forhandsbestilling-1/products/edda-pcb?variant=39481626001453)

Reviewer 2: ```suggestion
# Edda PCB

Alice layout PCB with top mounted USB C used in Mekanisk Edda

* Keyboard Maintainer: [martin](https://github.com/arnstadm)
* Hardware Supported: [Edda PCB](https://tastatur.no/collections/forhandsbestilling-1/products/edda-pcb?variant=39481626001453)
```
